cardInfoAPI-Level1.yaml

```diff
@@ -0,0 +1,1402 @@
+
+    CommonErrorResponse:
+      title: Common Error Response
+      description: General error response structure used in different response codes.
+      type: object
+      properties:
+        type:
+          $ref: '#/components/schemas/CommonErrorType'
+        title:
+          type: string
+          description: Short name for the problem.
+          examples:
+            - This is the general problem description
+        detail:
+          type: string
+          description: More details about the problem.
+          examples:
+            - Detailed problem description with respect to the current request
+        instance:
+          type: string
+          description: Reference to an individual entity or request which caused the problem.
+          examples:
+            - path/to/corresponding/resource
+
+    CommonErrorType:
+      title: Common Error Type
+      description: Error Types for CommonErrorResponse.
+      type: string
+      enum:
+        - /problems/INVALID_PAYLOAD
+        - /problems/MALFORMED_PAYLOAD
+        - /problems/INVALID_TOKEN
+        - /problems/EXPIRED_TOKEN
+        - /problems/INSUFFICIENT_PRIVILEGES
+        - /problems/NO_ACCESS_TO_RESOURCE
+        - /problems/RESOURCE_DOES_NOT_EXIST
+        - /problems/RESOURCE_NOT_READY
+        - /problems/RESOURCE_TOO_LARGE
+        - /problems/WRONG_METHOD
+        - /problems/OPERATION_NOT_ALLOWED
+        - /problems/TECHNICAL_ERROR
+        - /problems/NOT_IMPLEMENTED
+        - /problems/SERVICE_UNAVAILABLE
+      examples:
+        - /problems/TECHNICAL_ERROR
+
+  parameters:
+    path_cardId:
+      in: path
+      name: cardId
+      required: true
+      schema:
+        $ref: '#/components/schemas/CardId'
+      description: Identifier (technical) for a card for which information is requested.
+
+    path_tokenId:
+      in: path
+      name: tokenId
+      required: true
+      schema:
+        $ref: '#/components/schemas/TokenId'
+      description: Identifier (technical) for a token for which information is requested.
+
+    path_transactionId:
+      in: path
+      name: transactionId
+      required: true
+      schema:
+        $ref: '#/components/schemas/TransactionId'
+      description: Identifier (technical) for a transaction for which information is requested.
+
+    limit:
+      in: query
+      name: limit
+      required: false
+      schema:
+        type: integer
+        format: int32
+        minimum: 1
+        examples:
+          - 25
+      description: Optional parameter for pagination. Maximum number of entries to be returned.
+
+    offset:
+      in: query
+      name: offset
+      required: false
+      schema:
+        type: integer
+        format: int32
+        minimum: 0
+        examples:
+          - 25
+      description: Optional parameter for pagination. The number of items to skip before starting to collect the result set.
+
+    client_in_header:
+      in: header
+      name: X-CorAPI-Client-ID
```

Comment:
Link to the standardized headers is missing, that could explain difference from the `X-API-Key`?

Reply:
We are not sure what your comment means. What do you refer to with X-API-Key? It is not included in the endpoints. Do you suggest to add it?

Reply:
No additional input was received for the feedback so we don't change anything.